src/gpl/src/replace.tcl

```diff
@@ -37,6 +37,7 @@ sta::define_cmd_args "global_placement" {\
     [-timing_driven]\
     [-routability_driven]\
     [-incremental]\
+    [-force_gpu_acceleration]\
```

Comment:
Why do we have this and -gpu ?

Reply:
Sorry. I didn't clear it completely. We have changed this to "force_cpu".